src/juxt/site/alpha/graphql.clj

```diff
@@ -208,10 +227,19 @@
 
 (defn process-xt-results
   [field results]
-  (if (-> field ::g/type-ref ::g/list-type)
-    results
-    ;; If this isn't a list type, take the first
-    (first results)))
+  (let [type-ref (::g/type-ref field)]
+    (if-let [type (::g/non-null-type type-ref)]
+      (cond
+        (::g/list-type type)
+        (or results [])
```

Comment:
This isn't correct behavior for Non-Null types - see https://spec.graphql.org/June2018/#sec-Errors-and-Non-Nullability - I'll probably fix this in a future commit but would help to know if you had a particular requirement in mind.